00-framework-02-document-header.md

```diff
@@ -0,0 +1,157 @@
+# Document Header for libp2p Specs
+
+> A standard document header to indicate spec maturity, status & ownership
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Working Draft | Active | r0, 2019-05-27  |
+
+Authors: [@yusefnapora]
+
+Interest Group: TBD
+
+[@yusefnapora]: https://github.com/yusefnapora
+
+See the [lifecycle document][lifecycle-spec] for context about maturity level
+and spec status.
+
+[lifecycle-spec]: https://github.com/libp2p/specs/blob/master/00-framework-01-spec-lifecycle.md
+
+## Motivation
+
+The [maturity and lifecycle spec][lifecycle-spec] defines levels of maturity for
+libp2p specs, as well as the states that a spec can be in at a given time. It
+also introduces the notion of an `Interest Group`, which is a set of libp2p
+community members that have expressed interest in the spec and are willing to
+help move it forward in its evolution.
+
+This document defines a header format to convey this key status information in
+an easy-to-read manner.
+
+## Example 
+
+```markdown
+# Spec title
+
+> An optional one-liner summary of the spec
+
+| Lifecycle Stage | Maturity                 | Status | Latest Revision |
+|-----------------|--------------------------|--------|-----------------|
+| 2A              | Candidate Recommendation | Active | r0, 2019-05-27  |
+
+
+Authors: [@author1], [@author2]
+
+Interest Group: [@interested1], [@interested2]
+
+[@author1]: https://github.com/author1
+[@author2]: https://github.com/author2
+[@interested1]: https://github.com/interested1
+[@interested2]: https://github.com/interested2
```

Comment:
This looks redundant, the `@` mentions suffice.

Reply:
On github.com, yes, but that's not the only reading venue from these specs. If we remove the links, we'd being relying on a proprietary backend to fill in the blanks for us, which is something I don't want to do. People should be able to view these specs locally, in raw format, and follow the links manually. In the future, we'll want to compile them in a PDF book as well, or even a different site. Or export them to https://docs.libp2p.io.